todo_app/app.py

```diff
@@ -1,14 +1,46 @@
-from flask import Flask
-
+from flask import Flask, render_template, redirect, url_for, request
+from classes import to_do_item
+from trello import get_cards, post_item
 from todo_app.flask_config import Config
+import requests, os
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
-@app.route('/')
+@app.route('/', methods=['GET'])
 def index():
-    return 'Hello World!'
+    items = get_cards()
+    # app.logger.info('Processing get cards request')
+    return render_template('index.html', items = items)
+
+@app.route('/items/new', methods=['POST'])
+def add_item():
+    api_key = os.getenv('API_KEY')
+    api_token = os.getenv('API_TOKEN')
+    things_to_do = os.getenv('THINGS_TO_DO')
+    params = {'key': api_key, 'token': api_token, 'idList': things_to_do, 'name': request.form['title']}
```

Comment:
You could try reducing code duplication by creating a function to create a dictionary containing key and token.

Each request can then be simplified a bit, even if they need to add further params. E.g.
```
    params = get_trello_auth_params()
    params['name'] = request.form['title']
```